src/main/kotlin/org/kamiblue/client/module/modules/chat/FriendHighlight.kt

```diff
@@ -18,9 +18,9 @@ internal object FriendHighlight : Module(
     category = Category.CHAT,
     showOnArray = false
 ) {
-    private val bold = setting("Bold", true)
-    private val color = setting("Color", EnumTextColor.GRAY)
-    private val sound = setting("Sound", true)
+    private val bold = setting("Bold", true, description = "Display their name in bold")
+    private val color = setting("Color", EnumTextColor.GRAY, description = "Change the colour of their name")
+    private val sound = setting("Sound", true, description = "Play a sound")
```

Comment:
All of these are unnecessary, and `colour` is not spelled in line with the rest of KAMI Blue (American English)